doc/manual/src/contributing/hacking.md

```diff
@@ -68,12 +68,20 @@ To build Nix itself in this shell:
 To install it in `$(pwd)/outputs` and test it:
 
 ```console
+[nix-shell]$ make check
 [nix-shell]$ make install
 [nix-shell]$ make installcheck -j $NIX_BUILD_CORES
 [nix-shell]$ ./outputs/out/bin/nix --version
 nix (Nix) 3.0
 ```
 
+To iterate on for example `libexpr`, prioritizing the most relevant checks:
```

Comment:
What exactly do you mean by "iterate on"? What part of the example makes it "prioritise" the most relevant checks?

Reply:
The parts where the otherwise implied targets are mentioned: `libexpr-tests_RUN`, and because of the way the `make` invocations are ordered.

I wonder if merging some of the make invocations would improve parallelism without accidentally delaying the thing you want to test.